uber/configspec.ini

```diff
@@ -332,6 +332,12 @@ prereg_takedown = string(default="2016-01-18")
 # able to edit them after this happens.
 uber_takedown = string(default="2016-01-21")
 
+# On the last day of an event, we usually let people in for free when it's so close to
+# the end of the day that it would be silly to charge someone for e.g. an hour or two.
+# Badges registered through the at-the-door registration form after this time will
```

Comment:
Ack, good catch!  I swear I typed "marked as need not pay" but apparently not!  Sorry for the confusion, I'll push out an updated comment in just a minute.